syft/generic/frameworks/attributes.py

```diff
@@ -7,12 +7,26 @@
 
 from syft.generic.frameworks.hook.hook import FrameworkHook
 
+allowed_commands = set()
+
 
 class FrameworkAttributes(ABC):
+    _allowed_commands = None
+
     @abstractmethod
     def __init__(self, framework: ModuleType, hook: FrameworkHook):
         pass
 
+    @property
+    def allowed_commands(self):
+        if self._allowed_commands is None:
+            self._allowed_commands = allowed_commands
```

Comment:
Q: In Python this is doing copy of the alllowed_commands, right?

Reply:
I don't think so! But cant' be sure :/